plugin.rb

```diff
@@ -8,6 +8,7 @@
 # url: https://github.com/discourse/discourse-openid-connect
 
 enabled_site_setting :openid_connect_enabled
+enabled_site_setting :openid_connect_use_pkce
```

Comment:
`enabled_site_setting` means "when this site setting is disabled, turn off all plugin functionality". Each plugin can only have one. We definitely don't want the PKCE one here.
```suggestion
```